src/__tests__/helpers.ts

```diff
@@ -1,18 +1,31 @@
+import { MutatorFn } from 'src/serializers';
 import {
   Pipeline,
   Serializer,
   serializers as predefinedSerializers,
+  Step,
+  CommandStep,
 } from '../';
 import { resetUuidCounter } from './setup';
+import seedrandom from 'seedrandom';
 
 type SerializerType =
   | 'json'
   | 'json_depends_on'
   | 'yaml'
   | 'yaml_depends_on'
   | 'dot'
-  | 'structure';
+  | 'structure'
+  | 'yaml_mutate';
 
+const mutate: MutatorFn = async (entity: Step) => {
+  if (entity instanceof CommandStep) {
+    const seed = seedrandom(
```

Comment:
```suggestion
    const rand = seedrandom(
```

the seed is that you pass in as an argument